### DISCUSSION TITLE
How to fix this issue??

### DISCUSSION THREAD

**Discussion by BusyDragon-1237** [ROM-builders/temporary#12012] [Q&A]
FAILED: out/soong/.intermediates/vendor/dot/build/soong/generated_kernel_includes/gen/.dummy_dep
rm -rf out/soong/.intermediates/vendor/dot/build/soong/generated_kernel_includes/gen && out/soong/host/linux-x86/bin/sbox --sandbox-path out/soong/.temp --manifest out/soong/generator
The failing command was run inside an sbox sandbox in temporary directory
out/soong/.temp/sbox/e1a6bd15aff95db4844f2d49a91ebce2d4853be8
The failing command line was:
PATH=/home/cirrus/roms/DotOS-dot12.1/prebuilts/tools-dot/linux-x86/bin:/home/cirrus/roms/DotOS-dot12.1/prebuilts/build-tools/path/linux-x86:/home/cirrus/roms/DotOS-dot12.1/out/.path LD_LIBRARY_PATH=/home/cirrus/roms/DotOS-dot12.1/prebuilts/tools-dot/linux-x86/lib: PERL5LIB=/home/cirrus/roms/DotOS-dot12.1/prebuilts/tools-dot/common/perl-base BISON_PKGDATADIR=/home/cirrus/roms/DotOS-dot12.1/prebuilts/build-tools/common/bison /home/cirrus/roms/DotOS-dot12.1/prebuilts/build-tools/linux-x86/bin/make -j24 CFLAGS_MODULE="-fno-pic" CPATH="/usr/include:/usr/include/x86_64-linux-gnu" HOSTCFLAGS="-fuse-ld=lld" HOSTLDFLAGS="-L/usr/lib/x86_64-linux-gnu -L/usr/lib64 -fuse-ld=lld" DTC_EXT=/home/cirrus/roms/DotOS-dot12.1/prebuilts/misc/linux-x86/dtc/dtc LLVM=1 HOSTCC=/home/cirrus/roms/DotOS-dot12.1/prebuilts/clang/host/linux-x86/clang-r416183b1/bin/clang HOSTCXX=/home/cirrus/roms/DotOS-dot12.1/prebuilts/clang/host/linux-x86/clang-r416183b1/bin/clang++ LEX=/home/cirrus/roms/DotOS-dot12.1/prebuilts/build-tools/linux-x86/bin/flex YACC=/home/cirrus/roms/DotOS-dot12.1/prebuilts/build-tools/linux-x86/bin/bison M4=/home/cirrus/roms/DotOS-dot12.1/prebuilts/build-tools/linux-x86/bin/m4 -C  O=/home/cirrus/roms/DotOS-dot12.1/out/soong/.intermediates/vendor/dot/build/soong/generated_kernel_includes/gen ARCH=arm64 CROSS_COMPILE="/usr/bin/ccache /home/cirrus/roms/DotOS-dot12.1/prebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/bin/aarch64-linux-android-" CROSS_COMPILE_ARM32="/home/cirrus/roms/DotOS-dot12.1/prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.9/bin/arm-linux-androidkernel-" headers_install && touch out/soong/.temp/sbox/e1a6bd15aff95db4844f2d49a91ebce2d4853be8/out/.dummy_dep
make: Entering directory '/home/cirrus/roms/DotOS-dot12.1'
make: *** O=/home/cirrus/roms/DotOS-dot12.1/out/soong/.intermediates/vendor/dot/build/soong/generated_kernel_includes/gen: No such file or directory.  Stop.

this is the build link = https://cirrus-ci.com/task/4794814020452352?logs=build#L4455

**Comment by Apon77** [comment]
sorry, we don't support build-related logs. please ask your question here https://t.me/alaskalinuxuser_romdevelopment
or any other group 
thank you.
